Mathlib/FieldTheory/Finite/Basic.lean

```diff
@@ -619,6 +619,48 @@ theorem Int.ModEq.pow_card_sub_one_eq_one {p : ℕ} (hp : Nat.Prime p) {n : ℤ}
     · exact hpn.symm
   simpa [← ZMod.intCast_eq_intCast_iff] using ZMod.pow_card_sub_one_eq_one this
 
+theorem Int.prime_dvd_pow_card_sub_one {p : ℕ} (hp : Nat.Prime p) {n : ℤ} (hpn : IsCoprime n p) :
```

Comment:
```suggestion
theorem Int.prime_dvd_pow_sub_one {p : ℕ} (hp : Nat.Prime p) {n : ℤ} (hpn : IsCoprime n p) :
```
The spelling "card" here is weird... I think the "card" throughout the file is referring to the cardinality of the `ZMod` type, or at least about `ZMOD`, but this theorem has neither.